modules/intentIqAnalyticsAdapter.js

```diff
@@ -149,15 +146,19 @@ function bidWon(args) {
   if (iiqAnalyticsAnalyticsAdapter.initOptions.lsValueInitialized && !iiqAnalyticsAnalyticsAdapter.initOptions.lsIdsInitialized) { initReadLsIds(); }
   if (!iiqAnalyticsAnalyticsAdapter.initOptions.manualReport) {
```

Comment:
We need to ensure there’s no duplication. When `manualReport` is set to true, the report should **NOT** be sent automatically on the `bidWon` event, but only through `reportExternalWin`. When it's set to false, the report should be sent automatically on `bidWon`, and calling `reportExternalWin` should have no effect.